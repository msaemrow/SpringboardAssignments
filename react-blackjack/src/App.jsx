import { useState } from 'react'
import Game from "./components/Game"
import './App.css'

function App() {
  return (
    <div>
      <Game username="cardShark"/>
    </div>
  )
}

export default App;
