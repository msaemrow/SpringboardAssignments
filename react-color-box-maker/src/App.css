#root {
  margin: 10px auto;
  text-align: center;
  display: flex;
  flex-direction: column;
}
