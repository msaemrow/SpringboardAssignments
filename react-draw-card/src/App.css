body{
  background-color: green;
}

#root {
  margin: 0 auto;
  padding: 2rem;
  text-align: center;
}

.App-title{
  color: white;
}

