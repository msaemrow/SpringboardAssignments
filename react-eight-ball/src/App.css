body{
  background-color: rgb(188, 75, 237);
}

#root {
  max-width: 1280px;
  margin: 0 auto;
  text-align: center;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}

.App-header{
  margin:0 auto;
  font-size: 1.5rem;
}

.App-instructions{
  margin:5px auto;
  font-size: 0.9rem;
}
