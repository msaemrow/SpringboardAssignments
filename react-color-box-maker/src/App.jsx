import { useState } from 'react'
import './App.css'
import BoxList from "./components/BoxList"

function App() {
  const [count, setCount] = useState(0)

  return (
    <div>
      <h1>Create a color box!</h1>
      <BoxList />
    </div>

  )
}

export default App
