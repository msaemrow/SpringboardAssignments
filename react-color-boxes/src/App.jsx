import { React, useState } from 'react'
import BoxContainer from './components/BoxContainer';
import Box from "./components/Box";
import "./App.css"

function App() {
  return (
    <div>
      <BoxContainer />
    </div>
  )
}

export default App;
