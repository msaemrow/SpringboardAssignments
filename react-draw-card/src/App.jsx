import { useState } from 'react'
import reactLogo from './assets/react.svg'
import viteLogo from '/vite.svg'
import './App.css'
import Deck from "./components/Deck"
import Deckv2 from "./components/Deckv2"

// Deck is a manual card draw component
// Deckv2 is an automatic card draw component

function App() {
  return (
    <>
      <h1 className='App-title'>AUTOMATIC CARD DRAW</h1>
      <Deckv2 />
    </>
  )
}

export default App
