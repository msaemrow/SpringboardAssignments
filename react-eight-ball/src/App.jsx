import { useState } from 'react'
import reactLogo from './assets/react.svg'
import viteLogo from '/vite.svg'
import './App.css'
import answers from "./answer.json"
import EightBall from './components/EightBall'


function App() {
  return (
    <>
      <h1 className='App-header'>Magic 8 Ball!</h1>
      <p className='App-instructions'>Click the edge of the ball to reveal your answer</p>
      <EightBall answers={answers} />
    
    </>
  )
}

export default App
