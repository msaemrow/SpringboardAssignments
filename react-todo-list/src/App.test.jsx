import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import App from "./App"


describe('checks ToDo List componenet', () => {
  it('renders without crashing', () => {
    render(<App />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment).toMatchSnapshot();
  });
});
