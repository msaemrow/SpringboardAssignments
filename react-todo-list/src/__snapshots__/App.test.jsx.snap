// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`checks ToDo List componenet > matches snapshot 1`] = `[Function]`;
